modular_skyrat/modules/primitive_fun/code/big_mortar.dm

```diff
@@ -0,0 +1,115 @@
+/obj/structure/large_mortar
+	name = "large mortar"
+	desc = "A large bowl perfect for grinding and or juicing a large number of things at once."
+	icon = 'modular_skyrat/modules/primitive_fun/icons/cooking_structures.dmi'
+	icon_state = "big_mortar"
+	density = TRUE
+	anchored = TRUE
+	max_integrity = 100
+	pass_flags = PASSTABLE
+	custom_materials = list(/datum/material/wood = MINERAL_MATERIAL_AMOUNT * 10)
+	/// All of the items stored within this structure
+	var/list/contained_items
```

Comment:
I honestly don't know if you need this variable. You might be fine just looking into the contents?